kata-containers/kata-containers
# Fix async problems

- Fix fork/child in execute_hook
- Close stdin in execute_hook after it was sent
- Add unittest test_execute_hook
- Remove unnecessary #[async_trait]
- Remove crate signal-hook which are no longer used
- Fix unit test test_process
